src/components/index/Header.js

```diff
@@ -91,7 +91,6 @@ const Header = () => (
                     </h2>
 
                     <a className="btn" href="https://github.com/eclipse-theia/theia" target="_blank" rel="noopener noreferrer">View on GitHub</a>
-                    <a className="btn btn--cta" href="https://gitpod.io/#https://github.com/eclipse-theia/theia" target="_blank" rel="noopener">Try in Gitpod &nbsp;&nbsp;&rarr;</a>
                     <a className="btn btn--cta" href="/docs/blueprint_download/" rel="noopener">Try on Desktop &nbsp;&nbsp;&rarr;</a>
```

Comment:
I suggest to rename the button to "Try now &nbsp;&nbsp;&rarr;"

Reply:
done